reactor-test/src/main/java/reactor/test/scheduler/VirtualTimeScheduler.java

```diff
@@ -252,6 +254,7 @@ public Disposable schedule(Runnable task) {
 	}
 
 	@Override
+	@SuppressWarnings("deprecation")
```

Comment:
since `VirtualTimeScheduler` is public API, it should not suppress the warning. (I think it is the only public class that is suppressing the warning, but please double check)

Reply:
Good catch 👍 Removed the suppression